From 0b063c26d00ed38a6188033e64a0446004bfc665 Mon Sep 17 00:00:00 2001
From: Carlos Garcia Campos <cgarcia@igalia.com>
Date: Tue, 12 Sep 2023 11:11:09 +0200
Subject: [PATCH] Unreviewed. Disable frame rate throttling

It doesn't work with our display refresh implementation, so temporary
disable it until we find the right fix or a new display refresh monitor
implementation.

* Source/WebCore/platform/graphics/AnimationFrameRate.cpp:
---
 Source/WebCore/platform/graphics/AnimationFrameRate.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebCore/platform/graphics/AnimationFrameRate.cpp b/Source/WebCore/platform/graphics/AnimationFrameRate.cpp
index c341659636938..67468df09b18d 100644
--- a/Source/WebCore/platform/graphics/AnimationFrameRate.cpp
+++ b/Source/WebCore/platform/graphics/AnimationFrameRate.cpp
@@ -30,7 +30,7 @@
 
 namespace WebCore {
 
-static constexpr OptionSet<ThrottlingReason> halfSpeedThrottlingReasons { ThrottlingReason::LowPowerMode, ThrottlingReason::NonInteractedCrossOriginFrame, ThrottlingReason::VisuallyIdle };
+static constexpr OptionSet<ThrottlingReason> halfSpeedThrottlingReasons { };
 
 FramesPerSecond framesPerSecondNearestFullSpeed(FramesPerSecond nominalFramesPerSecond)
 {

diff --git a/Source/cmake/OptionsGTK.cmake b/Source/cmake/OptionsGTK.cmake
index fab244255665..50490d49e8b5 100644
--- a/Source/cmake/OptionsGTK.cmake
+++ b/Source/cmake/OptionsGTK.cmake
@@ -3,7 +3,7 @@ include(VersioningUtils)
 
 WEBKIT_OPTION_BEGIN()
 
-SET_PROJECT_VERSION(2 41 92)
+SET_PROJECT_VERSION(2 41 9292) # fake version to make it easy to detect if this patch is applied in a build
 
 # This is required because we use the DEPFILE argument to add_custom_command().
 # Remove after upgrading cmake_minimum_required() to 3.20.
